src/App.js

```diff
@@ -8,7 +9,15 @@ class App extends React.Component {
   }
 
   render() {
-    return <Calculator />;
+    return (
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<Home />} />
```

Comment:
- [ ] After debugging your code i do realize the home page by default is empty.  Kindly call your navbar between line number 14 and 15 so it can be like ` <NavBar />` do not forget to import it from the component like `import NavBar from './component/Navbar';` above.